packages/ast/src/clients/rpc/class/grpc-web/grpc-web.query.ts

```diff
@@ -303,9 +317,10 @@ export const createGrpcWebQueryClass = (
             return GrpcWebClassMethod(
                 context,
                 name,
-                key,
+                // key,
```

Comment:
please remove all commented code, we can always find them in history if we someday need them.

Reply:
yes, I removed it